### ISSUE TITLE
Tabs: add manual activation through keyboard navigation

### ISSUE THREAD

**Issue by imincheva** [IgniteUI/igniteui-angular#14839]
## Is your feature request related to a problem? Please describe.
In Web Components the Tabs have a property, which determines the keyboard navigation - whether it's manual or automatic, while in Angular only automatic keyboard navigation has been implemented

## Describe the solution you'd like
Manual keyboard navigation should be implemented in Angular as well. [Tabs Specification for Web Components](https://github.com/IgniteUI/igniteui-webcomponents/wiki/Tabs-Specification#igctabscomponent)
